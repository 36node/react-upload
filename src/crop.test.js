import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Crop from "./crop";

it("should renders correctly", () => {
  const tree = renderer.create(<Crop />).toJSON();
  expect(tree).toMatchSnapshot();
});
