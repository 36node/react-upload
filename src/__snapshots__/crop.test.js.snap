// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<div
  className="ReactCrop"
  onKeyDown={[Function]}
  onMouseDown={[Function]}
  onTouchStart={[Function]}
  role="presentation"
  tabIndex={1}
>
  <div>
    <img
      alt=""
      className="ReactCrop__image"
      onError={[Function]}
      onLoad={[Function]}
      src=""
    />
  </div>
</div>
`;
